Dockerfile.ubi

```diff
@@ -125,6 +125,22 @@ RUN --mount=type=cache,target=/root/.cache/ccache \
         CMAKE_BUILD_TYPE=Release \
         python3 setup.py bdist_wheel --dist-dir=dist
 
+#################### libsodium Build IMAGE ####################
+FROM base as libsodium-builder
+
+RUN microdnf install -y gcc gzip \
+    && microdnf clean all
+
+WORKDIR /usr/src/libsodium
+
+ARG LIBSODIUM_VERSION=1.0.19
```

Comment:
Is there a specific reason for using `1.0.19` instead of `1.0.20`?

Reply:
Uhh @tjohnson31415 did you explore this? I see the `1.0.20` was released before you created the PR in ibm

Reply:
Should be fine to upgrade. Initial integration testing I did was before 1.0.20 was released.

Reply:
Tried an image with `1.0.20`, the integration test works fine.